### DISCUSSION TITLE
Welcome to pekwm Discussions!

### DISCUSSION THREAD

**Discussion by pekdon** [pekwm/pekwm#93] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pclouds** [comment]
Cool! I miss mailing lists but I guess this will also work.

I'm a former Gentoo developer, former Git contributor, long time Linux user. I've switched to pekwm from fluxbox recently and basically just trying to scratch my itches (then convince pekdon to merge so that he bears the maintenance burden, muahaha, ok i'm kidding a bit).

There's one idea I'm thinking about but it's probably not fit in pekwm: I'm also a Scheme fan, Gauche Scheme specifically, so at some point I might fork pekwm to allow extensions (or even rewriting in Scheme). The Scheme community is super small though, this won't steal pekwm users.

**Comment by pigge2** [comment]
here's another old Gentoo, and long term pekwm user, these days i also run a few Artix (Arch) systems.. i don't think there's much more to say..

**Comment by okraits** [comment]
Hi guys,

I'm an old-time pekwm user who has been using other window managers (openbox, i3) for some years but returned to using pekwm last December.
I have also been using Arch Linux for more than a decade but then switched to Debian stable in 2016.

I try to contribute to pekwm and its community whenever I have the time, besides my work as a software developer and my family life.

**Comment by aznlir88** [comment]
Hello, years long Fluxbox user here. Saw pekwm is again in development - maybe it is worth trying out?

Just asked a question here about features we can enable/disable during compilation as I tend to do it with fluxbox - hopefully someone will provide some feedback. Personally I would like to be able not to compile in features I do not need and adjust program to my needs that way.

Probably will try pekwm as soon as I have some spare time to play around with it.

**Comment by r1w1s1** [comment]
Hi everyone,

I'm a long-time Slackware user, and I’ve spent years hopping between lightweight window managers like Fluxbox, IceWM, and even twm. Recently I decided to try pekwm, and after just a few days it already feels like home - fast, clean, and exactly the kind of minimalism I enjoy.

I use only a laptop screen (no external monitor) and prefer a simple, distraction-free setup. That’s also why I love Slackware and why I maintain a few SlackBuilds in my free time.

Happy to be here and looking forward to learning more about pekwm!
